### ISSUE TITLE
Feedback thread for the new patch

### ISSUE THREAD

**Issue by Grelo** [07th-mod/onikakushi#104]
Just tried the new patch for couple of minutes and it looks very good. With lip-sync the game looks outstanding and i can see lot of people just stop and say wow when comparing original release of Mangagamer to this mod. Good job.

As of bugs or errors i spotted a thin black line on the left side of the screen when hinamizawa overview started and adv textbox appeared at the bottom, not sure if itʼs fixable.
Lip-sync worked pretty fine most of the time, sometimes there was slight delay between voice and lip. Like half a second or even less.
But maybe because of my fast clicking iʼm not sure. 
And the lip-sync was turned off by default, i can see people struggle with this and post lot of comments "lip-sync not working". And maybe post somewhere that F10 is the key to setting window.
Will test more when i have time.

**Comment by ItaloKnox** [comment]
Fast clicking kinda breaks it, so that might be the case. Not sure how we are going to fix this now that @P-Chang left us. Also, as far as I know, the last thing he did was to post a new dll that has it enabled by default.

I'm worried about what we are going to do from now on. He pretty much left with all the code he wrote.

**Comment by Grelo** [comment]
We need to believe that he will change his mind in the near future. Is this affecting the release of the patch for other chapters? That would be sad :(

**Comment by ItaloKnox** [comment]
@enumag should know this better than me but I'm almost positive this completely delays every patch we have planned.

**Comment by Grelo** [comment]
@ItaloKnox Guy on steam pointed out that steam sprites link in the release page is down, maybe you can fix it.

**Comment by enumag** [comment]
We need to figure out a way how to hack lipsync into the other chapter dlls. Also some more work to match the voice analysis files to the right directories since none of us has P-Chang's PS3 voice analysis files, just PS2. I'm not sure how good or bad will that work.

So yeah. In the best case scenario it will just delay the releases. In bad case it might make the quaility lot worse than in Onikakushi. And in the worst case... well you get the idea.

Btw. @Grelo I was wondering if you could join our discord? Some of the development discussion kind of moved there.

**Comment by yagamisempai** [comment]
lip sync not work for me and i think its because of the f10 key you mentiones, sorry for post all of a sudden and here uwu

**Comment by ItaloKnox** [comment]
@Grelo Thanks for the heads up, I just goofed and forgot to change the extension.

**Comment by ItaloKnox** [comment]
@yagamisempai try closing and opening the game again. Don't forget to check if you have all the files, including the .dll. If I recall the hotkey to activate it was one of the keyboard numbers (those that are above the letters).

**Comment by yagamisempai** [comment]
i download the pacth, the voices (btw what do i do with the second folder in the voicearchive i think is a folder called spectrum or something) ang graphics, the graphics and voices work but there is no lip movement so i tough i did something wrong (sorry for my bad english)

**Comment by EliEron** [comment]
Are you talking about the PS3 voice analyze files P-chang posted in [this](https://github.com/07th-mod/higurashi-patch-compiler/issues/9#issuecomment-353749354) message?

If so then I have a copy of that archive. I don't have the PS2 voice analyze files he posted later though.

I have reuploaded the PS3 archive here: http://www.mediafire.com/file/ipcuwdye213ncrw/PS3VoiceAnalyze.zip

Also just so you know one of the C# decompilers that P-Chang mentioned using for patching in one of his early posts [dnSpy](https://github.com/0xd4d/dnSpy), has the ability to modify C# Assemblies directly. Meaning you can edit and add methods/classes to the Assembly-CSharp.dll file without having to recompile it.

If you guys don't manage to get the decompiled source to compile (I never figured out how to do it) then that program might come in handy.

**Comment by ItaloKnox** [comment]
Thank you so much, the PS3 files pretty much saved the lipsync. I will also be checking the tool you posted, but I think you're right on track. Anyway, I can't thank you enough for this.

Also, you're right, it's the 7 key. I had an issue a couple days ago where it didn't work, so I had to go back and forth the menu clicking everything so it could trigger the hotkey. Not sure what happened, but might be what @yagamisempai is experiencing.

**Comment by yagamisempai** [comment]
what do i have to do with the spectrum folder in the voice archives?
because i just put the voice folden in to my onikakushi game and the voices work perfectly but i wonder if i need the other one for something?
sorry for the bother again

**Comment by ItaloKnox** [comment]
The lipsync needs that folder to work. Just extract it alongside the voices folder.

**Comment by yagamisempai** [comment]
![sin titulo](https://user-images.githubusercontent.com/28794338/35897069-ba8cf838-0bbf-11e8-877c-f38652cdca27.png)

i make it work but now i cant get rid of that screen uwu

**Comment by Grelo** [comment]
@yagamisempai Try F10 again.

**Comment by yagamisempai** [comment]
thanks a lot

**Comment by enumag** [comment]
@EliEron Wow that's awesome! We had the PS2 archive and was missing the PS3. This is great, thank you very much!! Now we have everything.

@yagamisempai The spectrum directory is not necessarily required for LipSync (the game has a fallback if spectrum files are not available) but it makes it significantly better. It synchronizes the mouth movement with pauses in the voice.

**Comment by enumag** [comment]
@EliEron Yep, we're missing that too. Get it up here! 👏

**Comment by EliEron** [comment]
Alright, here is the original archive he posted:
http://www.mediafire.com/file/rxtn44dw9mff62x/chapter2dlls_v3.x.x.rar

That dll is setup to read the voice analysis data from the voice folder. Here is a patched dll that reads it from the spectrum folder just like the current Onikakushi dll:
http://www.mediafire.com/file/3jo7o9p5os6w62k/Assembly-CSharp-spectrum.zip

I patched the file path in the dll using the [dnSpy](https://github.com/0xd4d/dnSpy) tool.

**Comment by temp-user527** [comment]
P-Chang left? That's a shame, I really owe you guys and P-Chang did really good work.

As for the lip-syncing, did a clean install, installed spectrum files and enabled the setting in the config. 
No luck on Linux, will try Windows when I'm at my main computer and see if I can narrow the issue down.

**Comment by enumag** [comment]
@temp-user527 Unfortunately yes, he did. But at this point it seems we'll be able to continue without him just fine, if with some delay from the original plan.

Thanks for trying. Please let us know what you can find out.

**Comment by enumag** [comment]
@EliEron Forgot to answer here. Thank you very much. I think Italo already used your fixed version when making the diffs for us.

**Comment by temp-user527** [comment]
I tried the patch out on both Windows and Linux and it works for me.
I think my saves for the game were a little bit wonky because I had run the unmodded game after playing the mod.
I don't think too many users will run into issues if it's their first time playing or anything.

**Comment by enumag** [comment]
@temp-user527 So it works on Linux actually and it was a problem of using old save files?

**Comment by temp-user527** [comment]
@enumag Works now. 
I had done clean install a few times but had no luck. Then I remembered one time where the game was stuck on fullscreen and I had to delete a dat file. For lip-sync, I just went ahead and wiped all the save and config files just to see if it would work and it worked.
I don't think the save files were the issue but perhaps one of the config files got bugged out for me because of how I a switched back and forth between modded and unmodded... I don't think it will be a common or significant issue.
